submissions/YuraZagor/hooliPopup/style.css

```diff
@@ -0,0 +1,140 @@
+* {
+	box-sizing: border-box;
+	position: relative;
+	margin: 0;
+	padding: 0;
+}
+
+.ul--menu, .popups--container, .popups--container--all {
+	list-style-type: none;	
+}
+
+.ul--menu {
+	display:flex;
+	justify-content: flex-end;
+	align-items: center;
+	
+}
+.li--menu {
+	padding: 10px;	
+}
+
+.popup--item {
+	width: 2.1em;
+	height: 2.1em;
+	cursor: pointer;
+	object-fit: contain;
+	padding: 2px;
+	margin: 10px;
+	user-select: none;
+}
+
+.sub-menu .menu--img {
+	width: 1.1em;
+	height: auto;
+	cursor: pointer;
+}
+
+.outer-popup-container{
+	display: none;
+	position: absolute;
+	top: 45px;
+	right: -80px;
+}
+
+.inner-container{
+	max-height: 250px;
+	border: 0.5px solid lightcoral;
+	border-radius: 6px;
+	justify-items: center;
+	align-items: center;
+	scroll-behavior: auto;
+	overflow-y:scroll;
+	margin: 0 10px;
+	padding: 0 10px;
+}
+
+.popups--container{
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;	
+}
+
+.popups--container--all{
+	display: none;
+	grid-template-columns: 1fr 1fr 1fr;	
+}
+
+.show--all, .checkbox-icon {
+	position: absolute;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	appearance: none;		
+}
+
+/* .checkbox-icon {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	overflow:hidden;
+	clip: rect(0 0 0 0);
+} */
```

Comment:
Avoid commented code in your work

Reply:
cleaned the working mess, sorry abt that